yt/geometry/oct_container.pxd

```diff
@@ -27,12 +27,6 @@ cdef struct OctKey:
     np.int64_t *indices
     np.int64_t pcount
 
-cdef struct OctInfo:
```

Comment:
This is due to a circular dependency issue between `oct_container.p[yx|xd]` and `oct_visitor.p[yx|xd]`, the former being dependent on the latter, so `OctInfo` needs to be in `oct_visitor` instead.

This was not necessary before, because `OctInfo` wasn't used in `oct_visitor` but it is now.

If you know how to solve that more gracefully, I would be glad to implement it!